lib/faraday/adapter/em_synchrony.rb

```diff
@@ -26,6 +26,11 @@ def call(env)
         # Queue requests for parallel execution.
         if env[:parallel_manager]
           env[:parallel_manager].add(request, http_method, request_config(env)) do |resp|
+            if (req = env[:request]).stream_response?
```

Comment:
nope ... could put it on the request or make it static ...

On Tue, Aug 30, 2016 at 3:11 PM, Martin Mauch <notifications@github.com>
wrote:

> In lib/faraday/adapter/em_synchrony.rb
> <https://github.com/lostisland/faraday/pull/604#discussion_r76889487>:
>
> > @@ -26,6 +26,11 @@ def call(env)
> >          # Queue requests for parallel execution.
> >          if env[:parallel_manager]
> >            env[:parallel_manager].add(request, http_method, request_config(env)) do |resp|
> > +            if (req = env[:request]).stream_response?
>
> I'm running in circles trying to find a nice way to extract a method here.
> Do you have something in mind?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/lostisland/faraday/pull/604/files/60cdd853545a06a19fd9cd771bbace1196cea141#r76889487>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAAsZz0RQFxpvFbCQALEXeVix6izqDMnks5qlKp7gaJpZM4JxAhc>
> .
>